---
uint8[] graph_data_buffer
int64[] cover_point_ind_to_graph_ind
